spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-configuration-watcher-it/src/test/java/org/springframework/cloud/kubernetes/configuration/watcher/ActuatorRefreshIT.java

```diff
@@ -49,66 +50,79 @@
 /**
  * @author Ryan Baxter
  */
-public class ActuatorRefreshIT {
-
-	private static final String CONFIG_WATCHER_WIREMOCK_DEPLOYMENT_NAME = "config-watcher-wiremock-deployment";
-
-	private static final String CONFIG_WATCHER_WIREMOCK_APP_NAME = "config-watcher-wiremock";
```

Comment:
this test has slightly eaten my soul :) it took a long time to understand how it is _really_ set-up and understand that we need to bind static ports to testscontainers, basically like docker does it via : `docker ... -p 8080:8080`. The default mode in which testcontainers functions is that it assigns a random port to what is exposed. So if you expose something on 8080 (then tell testcontainers about it via `.withExposedPorts`), testcontainers will create a random port (55555 for example), that you can find out about via : `....getMappedPort(8080); // will return 55555` and _then_ use that `55555`.

Because of the way our tests are set-up, this is not an option for us, so we need to statically bind these ports.